src/examples/mailchimp/src/backend/schema/index.ts

```diff
@@ -0,0 +1,56 @@
+import { MailingListResolver, createMailingListResolver } from './list';
+import { CategoryResolver, createCategoryResolver } from './category';
+import { InterestResolver, createInterestResolver } from './interest';
+import { MemberResolver, createMemberResolver } from './member';
+
+export type ClientOptions = {
+	apiKey: string;
+	server: string;
+};
+
+export const mailingList: MailingListResolver =
+	process.env.MAILCHIMP_API_KEY &&
```

Comment:
Would be good to try to catch any of these missing env vars with a specific error.

Reply:
added check for missing environment variables